source/beginners_guide/raw_beginners_guide.rst

```diff
@@ -184,7 +184,62 @@ Supported Modsets
       :class-title: header-3-light
       :class-container: sd-card-3
 
-      We are currently supporting more additional mods, details will be following soon.
+      Antistasi has built in compatibility for the following mods. If loaded (and if applicable) the needed items of the mods will either be available in the arsenal, can be found on AI or in lootboxes. This of course also is depending on which item it is.
+      In addition there are parameters which might influence that spawning behaviour so make sure you check them out.
+
+      .. rst-class:: table-additional-mods
+
+      .. flat-table::
+         :header-rows: 0
+         :widths: 20 80
+
+         *  - `CBA <https://steamcommunity.com/workshop/filedetails/?id=450814997>`_
+            - CBA is a framework which adds a range of features Arma in itself does not supply. This mod is a dependency for many other mods and it also adds some additional functionality within Antistasi, for example the Extended Debug Console.
```

Comment:
I'm not actually sure if the debug console is affected anymore, since we're using Jeroen's extended debug console. 

Main thing that CBA adds to Antistasi is the ability to modify garage settings.

Reply:
Jeroens Extended Debug Console requires CBA.
Will adapt the text though.
Here is 3.0 loaded without CBA
![image](https://user-images.githubusercontent.com/50651578/210189528-4ffeef11-8815-4f1d-91bf-e3efd72d0398.png)